src/import-data/useImportRequest.ts

```diff
@@ -156,18 +182,26 @@ export const getImportRequest = (queryParams: QueryParams): Promise<ImportReques
 };
 
 export type UseImportRequestResult =
-  | { status: 'Loading' }
+  | { status: 'Loading'; message?: string }
   | { status: 'Ready'; importRequest: ImportRequest }
   | { status: 'Error'; error: Error };
 
+/**
+ * For each format that takes a long time to prepare, define a message to display while the import request is being loaded.
+ */
+const messages = {
+  tdrexport: 'Preparing your snapshot for importing.',
```

Comment:
Maybe, Preparing your snapshot for import?

Reply:
I don't have a strong opinion and am not sure if there's a terra-ui standard on language in messages like this. If you think `Preparing your snapshot for import` reads better I can change it. I guess `import` does sound a bit more formal than `importing`.